src/main/webapp/i18n/de/instructorExerciseDashboard.json

```diff
@@ -1,8 +1,8 @@
 {
     "artemisApp": {
         "instructorExerciseDashboard": {
-            "title": "Instructor Übungs-Dashboard",
-            "pageHeader": "Instructor Übungs-Dashboard für",
+            "title": "Dozenten Dashboard für Übung",
+            "pageHeader": "Dozenten Dashboard für Übung",
             "back": "Zurück",
             "noProblemStatement": "No problem statement available",
```

Comment:
While we're here: Shouldn't this be German as well? (`Keine Problembeschreibung verfügbar`)